src/index.html

```diff
@@ -51,7 +51,10 @@ <h1 class="homepage__hero__text">Publish <span class="color-white">mobile, web,
             <div class="col-md-12 col-lg-8 offset-lg-2">
                 <span class="homepage__intro__partner">Made by {% asset homepage/logo-google.svg alt='Google logo' %}</span>
                 <p class="homepage__intro__statement text-center">
-                    Flutter is Google’s portable UI toolkit for building beautiful, natively-compiled applications for <a href="https://flutter.dev/docs">mobile</a>, <a href="https://flutter.dev/web">web</a>, and <a href="https://flutter.dev/desktop">desktop</a> from a single codebase.
+                    Flutter is Google’s portable UI toolkit for building
```

Comment:
Here's some more center-aligned p text. Do we intentionally do this in some cases? 

<img width="897" alt="Screen Shot 2019-06-14 at 2 13 19 PM" src="https://user-images.githubusercontent.com/5550175/59538495-d45f9b80-8eae-11e9-8a69-ea72222b7aa1.png">

Reply:
Definitely. This page was designed by a UX designer with Martin's input.